src/main/java/com/thebois/ColonyManagement.java

```diff
@@ -59,12 +66,19 @@ public void create() {
 
         setUpUserInterfaceSkin();
 
-        setUpModelViewController(tileSize);
-
+        // Model
+        createModels();
+        // Views
+        createGameView(tileSize);
+        createInfoView();
         // Screens
         gameScreen = new GameScreen(VIEWPORT_HEIGHT, VIEWPORT_WIDTH, uiSkin, gameView, infoView);
-
+        projector = gameScreen.getProjector();
```

Comment:
Feels out of place. either this line where its getting used. Or just call the getter when we need it (because I only see this being used once?)